code/modules/reagents/chemistry/holder.dm

```diff
@@ -933,7 +933,10 @@
 					break
 				total_matching_catalysts++
 			if(cached_my_atom)
-				matching_container = reaction.required_container ? (cached_my_atom.type == reaction.required_container) : TRUE
+				if(reaction.required_container_accepts_subtypes)
+					matching_container = reaction.required_container ? (istype(cached_my_atom, reaction.required_container)) : TRUE
+				else
+					matching_container = reaction.required_container ? (cached_my_atom.type == reaction.required_container) : TRUE
```

Comment:
Change to `!reaction.required_container || other_condition`

Reply:
What did they mean by this

Reply:
What part doesn't make sense

Reply:
This makes a lot more sense reading it two days later